package.json

```diff
@@ -15,60 +15,61 @@
     "generate:version": "node bin/generate-version.js",
     "lint": "eslint --ext .js,.ts --ignore-path .gitignore --max-warnings 0 .",
     "lint:fix": "eslint --ext .js,.ts --ignore-path .gitignore --max-warnings 0 --fix .",
-    "test:dts": "tsc --noEmit --isolatedModules dist/fingerprintjs-pro-angular/fingerprintjs-fingerprintjs-pro-angular.d.ts",
+    "test:dts": "tsc --noEmit --isolatedModules dist/fingerprintjs-pro-angular/index.d.ts",
     "test": "jest",
     "test:coverage": "jest --coverage",
     "docs": "typedoc projects/fingerprintjs-pro-angular/src/public-api.ts --out docs"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~13.3.0",
-    "@angular/common": "~13.3.0",
-    "@angular/compiler": "~13.3.0",
-    "@angular/core": "~13.3.0",
-    "@angular/forms": "~13.3.0",
-    "@angular/platform-browser": "~13.3.0",
-    "@angular/platform-browser-dynamic": "~13.3.0",
-    "@angular/platform-server": "~13.3.0",
-    "@angular/router": "~13.3.0",
-    "@nguniversal/express-engine": "13.1.1",
+    "@angular/animations": "^15.2.10",
+    "@angular/common": "^15.2.10",
+    "@angular/compiler": "^15.2.10",
+    "@angular/core": "^15.2.10",
+    "@angular/forms": "^15.2.10",
+    "@angular/platform-browser": "^15.2.10",
+    "@angular/platform-browser-dynamic": "^15.2.10",
+    "@angular/platform-server": "^15.2.10",
+    "@angular/router": "^15.2.10",
+    "@fingerprintjs/fingerprintjs-pro-spa": "^1.3.0",
+    "@nguniversal/express-engine": "^15.2.1",
     "express": "^4.15.2",
     "rxjs": "~7.5.0",
     "tslib": "^2.3.0",
     "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~13.3.6",
-    "@angular-eslint/schematics": "^13.2.1",
-    "@angular/cli": "~13.3.6",
-    "@angular/compiler-cli": "~13.3.0",
+    "@angular-devkit/build-angular": "^15.2.10",
+    "@angular-eslint/schematics": "^15.2.1",
+    "@angular/cli": "^15.2.10",
+    "@angular/compiler-cli": "^15.2.10",
     "@commitlint/cli": "^17.2.0",
-    "@commitlint/config-conventional": "^17.2.0",
-    "@nguniversal/builders": "^13.1.1",
+    "@fingerprintjs/commit-lint-dx-team": "^0.0.2",
+    "@fingerprintjs/conventional-changelog-dx-team": "^0.1.0",
+    "@fingerprintjs/eslint-config-dx-team": "^0.1.0",
+    "@fingerprintjs/prettier-config-dx-team": "^0.1.0",
+    "@nguniversal/builders": "^15.2.1",
     "@types/express": "^4.17.0",
-    "@types/jest": "^27.5.1",
+    "@types/jest": "^29.5.12",
     "@types/node": "^12.11.1",
-    "@typescript-eslint/eslint-plugin": "^5.27.0",
-    "@typescript-eslint/parser": "^5.27.0",
+    "@typescript-eslint/eslint-plugin": "^6.21.0",
+    "@typescript-eslint/parser": "^6.21.0",
     "commitizen": "^4.2.5",
-    "conventional-changelog-conventionalcommits": "^5.0.0",
     "cz-conventional-changelog": "^3.3.0",
     "eslint": "^8.16.0",
```

Comment:
`eslint` is a dependency of a `"@fingerprintjs/eslint-config-dx-team"` too to simplify version update and keep all packages compatible